backend/functions/lib/use-cases/orders/orders.ts

```diff
@@ -342,24 +343,22 @@ export class OrdersUseCase {
 
     const createdConsolidation = await this.consolidationsRepo.put(context, newConsolidation);
     response.push(createdConsolidation);
-    const { docketEntries: _docketEntries, ...leadCaseSummary } = leadCase;
+
+    const assignmentUseCase = new CaseAssignmentUseCase(context);
```

Comment:
Recommend a factory pattern here

Reply:
What would that do for us? The factory pattern is best used when dealing with abstract classes with multiple concrete implementations.